tests/integration/test_record_mode.py

```diff
@@ -9,16 +8,21 @@ def test_once_record_mode(tmpdir):
     with vcr.use_cassette(testfile, record_mode="once"):
         # cassette file doesn't exist, so create.
         response = urlopen('http://httpbin.org/').read()
```

Comment:
should I also change lines like this to `urlopen('http://httpbin.org/').read()` , and remove the `assert response` ?

(It is very late here, so I'll pick this up tomorrow and revise it to better suit your tastes.)